publish.js

```diff
@@ -0,0 +1,272 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/fluid-project/first-discovery-server/raw/master/LICENSE.txt
+*/
+
+"use strict";
+
+var publish = {};
+var pkg = require("./package.json");
+publish.execSync = require("child_process").execSync;
```

Comment:
I think var child_process = require("child_process") and then child_process.execSync() is clearer - no point in trying to export this standard node API